### PR TITLE
spring-jms

### PR DESCRIPTION
**Pull request by kirangonella**
Hi,

Updated indentation, added JUnit test scripts and modified the way ActiveMQ is called

### CODE REVIEW

File: spring-jms/src/test/com/baeldung/spring/jms/DefaultTextMessageSenderTest.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import com.baeldung.spring.jms.SampleJmsMessageSender;
+
+public class DefaultTextMessageSenderTest {
+
+    private SampleJmsMessageSender messageProducer;
+
+    @SuppressWarnings("resource")
+    @Before
```
Comment (pivovarit) [inline spring-jms/src/test/com/baeldung/spring/jms/DefaultTextMessageSenderTest.java]: "@Before" runs before every method. "@BeforeClass" runs once before running all tests. I guess we need to second option in this case

File: spring-jms/src/test/com/baeldung/spring/jms/MapMessageConvertAndSendTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import com.baeldung.spring.jms.Employee;
+import com.baeldung.spring.jms.SampleJmsMessageSender;
+
+public class MapMessageConvertAndSendTest {
+
+    private SampleJmsMessageSender messageProducer;
+
+    @SuppressWarnings("resource")
+    @Before
```
Comment (pivovarit) [inline spring-jms/src/test/com/baeldung/spring/jms/MapMessageConvertAndSendTest.java]: @BeforeClass just like above

**Comment by kirangonella** [comment]
Changed as suggested. Thank you.

**Comment by kirangonella** [comment]
Hi Grzegorz Piwowarek,

Changed as suggested. Thank you.

Regards,
Kiran.

On Sun, Oct 9, 2016 at 2:14 PM, Grzegorz Piwowarek <notifications@github.com

**Comment by pivovarit** [comment]
@kirangonella I just noticed that there are some conflicting files. Please resolve those conflicts and we can merge

**Comment by kirangonella** [comment]
Hi Grzegorz Piwowarek,

You can approve those changes as I am unable to do so.

I updated those files as well.

Thank you,

Regards,
Kiran.

On Sun, Oct 9, 2016 at 2:26 PM, Grzegorz Piwowarek <notifications@github.com

**Pull request merged**